octobot_services/service_feeds/service_feed_factory.py

```diff
@@ -18,17 +18,26 @@
 from octobot_commons.logging.logging_util import get_logger
 
 from octobot_services.service_feeds.abstract_service_feed import AbstractServiceFeed
+from octobot_services.service_feeds.service_feeds import ServiceFeeds
 
 
 class ServiceFeedFactory:
-    def __init__(self, config, main_async_loop):
+    def __init__(self, config, main_async_loop, bot_id):
         self.logger = get_logger(self.__class__.__name__)
         self.config = config
         self.main_async_loop = main_async_loop
+        self.bot_id = bot_id
 
     @staticmethod
-    def get_available_service_feeds() -> list:
-        return get_all_classes_from_parent(AbstractServiceFeed)
+    def get_available_service_feeds(in_backtesting: bool) -> list:
+        feeds = get_all_classes_from_parent(AbstractServiceFeed)
+        if in_backtesting:
```

Comment:
What will be the use of `in_backtesting` param ?

Reply:
To know if the service feeds to be created should be the backtesting versions of the actual service feeds or the regular version

Reply:
Oh yes you're  right